### PR TITLE
Add best practices guide for live process monitors

### PR DESCRIPTION
**Pull request by jhgilbert**
### What does this PR do? What is the motivation?
DOCS-7086

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/monitors/guide/monitor-a-live-process.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+An improperly configured monitor is prone to false positives. This guide covers the recommended best practices for creating a reliable live process monitor. For a detailed overview of the monitor creation process, see [Live process monitor creation][3].
+
+## Best practices
+
+### Scoping the monitor
+
+To avoid false positives, the scope of your monitor should not exceed a few thousand processes. Because text search is fuzzy, tags are the most accurate way to adjust the scope of your monitor.
+
+#### Example workflow
```
Comment (brett0000FF) [inline content/en/monitors/guide/monitor-a-live-process.md:23]:
```suggestion
Example workflow:
```
Not sure if we need a full on heading here.

File: content/en/monitors/guide/monitor-a-live-process.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+1. Navigate to the [**Monitors > New Monitor > Live Process**][1] page.
+
+1. Add tags to the monitor in the **by tags** field.
```
Comment (brett0000FF) [inline content/en/monitors/guide/monitor-a-live-process.md:27]:
```suggestion
2. Add tags to the monitor in the **by tags** field. 
```
This renders correctly, but just fixing to help us when reading the source.

**Review by brett0000FF — APPROVED** [review]
This looks great!

File: content/en/monitors/guide/monitor-a-live-process.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+2. Optionally, refine the monitor's scope by adding search text to the **by text** field. In the example below, only processes matching `cluster worker` are included.
+
+{{< img src="monitors/monitor_types/process/text-scoped-process-monitor.png" alt="A live process monitor that has been scoped using text search" style="width:100%;" >}}
+
+3. If your monitor's scope still exceeds a few thousand processes total across all process groups, use additional tags to break it into multiple monitors. 
+    - For example, you can use the `env` tag to create separate monitors for `prod` and `staging`.
+
+### Choosing a time frame
+
+A common misconception is that increasing the evaluation window leads to slower responses or missed alerts, but a monitor continuously assesses data no matter what query evaluation interval you choose. The evaluation interval only determines how many data points are used to decide whether an anomaly exists.
```
[the remark this answers is not in the document]
Comment (jhgilbert) [reply content/en/monitors/guide/monitor-a-live-process.md:41]: TIL!

File: content/en/monitors/guide/_index.md [hunk 4]
```diff
@@ -25,6 +25,7 @@ cascade:
     {{< nextlink href="monitors/guide/adjusting-no-data-alerts-for-metric-monitors" >}}Adjusting "No Data" alerts for metric Monitors{{< /nextlink >}}
     {{< nextlink href="monitors/guide/prevent-alerts-from-monitors-that-were-in-downtime" >}}Prevent alerts from monitors that were in downtime{{< /nextlink >}}
     {{< nextlink href="monitors/guide/reduce-alert-flapping" >}}Reduce alert flapping{{< /nextlink >}}
+    {{< nextlink href="monitors/guide/monitor-a-live-process" >}}Monitor a live process{{< /nextlink >}}
```
Comment (just-chillin) [inline content/en/monitors/guide/_index.md:28]: I feel as if "Monitor a live process" sounds a bit strange. What about "Best Practices for Live Process Monitoring"?

File: content/en/monitors/guide/monitor-a-live-process.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    - For example, you can use the `env` tag to create separate monitors for `prod` and `staging`.
+
+### Choosing a time frame
+
+A common misconception is that increasing the evaluation window leads to slower responses or missed alerts, but a monitor continuously assesses data no matter what query evaluation interval you choose. The evaluation interval only determines how many datapoints are used to decide whether an anomaly exists.
+
+By increasing the evaluation window, you can ensure that you are only alerted if a behavior is happening consistently, not temporarily.
+
+- To avoid false positives, use a minimum interval of **5 minutes**.
+- If your monitor uses tags that come from a cloud provider crawler, use a minimum interval of **15 minutes**.
```
Comment (just-chillin) [inline content/en/monitors/guide/monitor-a-live-process.md:46]:
```suggestion
- If your monitor uses tags that come from a cloud provider integration, use a minimum interval of **15 minutes**.
```
I'm not sure if customers know what a crawler is?

**Review by mbotarro — COMMENTED** [review]
I left some small comments but this looks great! Thanks for putting up this doc!

File: content/en/monitors/guide/_index.md [hunk 6]
```diff
@@ -25,7 +25,8 @@ cascade:
     {{< nextlink href="monitors/guide/adjusting-no-data-alerts-for-metric-monitors" >}}Adjusting "No Data" alerts for metric Monitors{{< /nextlink >}}
     {{< nextlink href="monitors/guide/prevent-alerts-from-monitors-that-were-in-downtime" >}}Prevent alerts from monitors that were in downtime{{< /nextlink >}}
     {{< nextlink href="monitors/guide/reduce-alert-flapping" >}}Reduce alert flapping{{< /nextlink >}}
-    {{< nextlink href="monitors/guide/monitor-a-live-process" >}}Monitor a live process{{< /nextlink >}}
+    {{< nextlink href="monitors/guide/monitoring-sparse-metrics" >}}Monitoring Sparse Metrics{{< /nextlink >}}
+    {{< nextlink href="monitors/guide/best-practices-for-live-process-monitoring" >}}Best practices for live process monitoring{{< /nextlink >}}
```
Comment (mbotarro) [inline content/en/monitors/guide/_index.md:29]: Should `Live Process Monitoring` be capitalised here?
```suggestion
    {{< nextlink href="monitors/guide/best-practices-for-live-process-monitoring" >}}Best practices for Live Process Monitoring{{< /nextlink >}}
```

**Review by mbotarro — APPROVED** [review]
Thanks for addressing all the comments, Jen!

**Review by vatteh — APPROVED** [review]

**Pull request merged**